### PR TITLE
[Bug] Check whitelisting when connection

### PR DESCRIPTION
**Pull request by minsulee2**
- Check PEER_WHITELIST when setting peer candidates
- Update some logs

### CODE REVIEW

**Comment by minsulee2** [comment]
Never mind the test! Deployment broke again

File: p2p/index.js [hunk 1]
```diff
@@ -671,8 +678,14 @@ class P2pClient {
     }
   }
 
-  setPeerCandidate(jsonRpcUrl, queriedAt) {
-    this.peerCandidates.set(jsonRpcUrl, { queriedAt });
+  setPeerCandidate(jsonRpcUrl, address, queriedAt) {
+    if (CommonUtil.isArray(NodeConfigs.PEER_WHITELIST) &&
+        NodeConfigs.PEER_WHITELIST.includes(address)) {
+      this.peerCandidates.set(jsonRpcUrl, { queriedAt });
+    } else if (CommonUtil.isString(NodeConfigs.PEER_WHITELIST) &&
```
Comment (platfowner) [inline p2p/index.js:685]: minor) can be merged into one?
e.g. `CommonUtil.isWildcard() || (CommonUtil.isArray() && NodeConfigs.PEER_WHITELIST.includes()) {`

File: p2p/server.js [hunk 2]
```diff
@@ -92,7 +92,9 @@ class P2pServer {
           NodeConfigs.MAX_NUM_INBOUND_CONNECTION) {
         this.setServerSidePeerEventHandlers(socket, url);
       } else {
-        logger.error(`Cannot exceed max connection: ${NodeConfigs.MAX_NUM_INBOUND_CONNECTION}`);
+        logger.error(`Cannot exceed max connection: ${NodeConfigs.MAX_NUM_INBOUND_CONNECTION}\n` +
```
Comment (platfowner) [inline p2p/server.js:95]: what about just 'info' level? if it's a temporary level for intensive debugging, could you add a todo?

File: p2p/server.js [hunk 3]
```diff
@@ -453,7 +455,7 @@ class P2pServer {
             } else {
               const addressFromSig = P2pUtil.getAddressFromMessage(parsedMessage);
               if (!P2pUtil.checkPeerWhitelist(addressFromSig)) {
-                logger.debug(`This peer(${addressFromSig}) is not on the PEER_WHITELIST.`);
+                logger.error(`This peer(${addressFromSig}) is not on the PEER_WHITELIST.`);
```
Comment (platfowner) [inline p2p/server.js:458]: ditto?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks for the quick delivery!

File: p2p/index.js (hunk 1 above)
Comment (minsulee2) [reply p2p/index.js]: Oh, My brain was not working! Thanks!

File: p2p/server.js (hunk 2 above)
Comment (minsulee2) [reply p2p/server.js]: Done!

File: p2p/server.js (hunk 3 above)
Comment (minsulee2) [reply p2p/server.js]: Done!

**Comment by minsulee2** [comment]
Thanks for the fast review!! Merging now for the next expr.

**Pull request merged**